### PR TITLE
[Project] Establish a Lyft support rotation

### PR DESCRIPTION
**Pull request by jpsim**
This change includes:

* Documentation stating the purpose, function and reasoning behind the Lyft support rotation.
* A YAML file capturing the rotation list of GitHub user names and who's currently on rotation.
* A script to bump who's on rotation to the next person.
* A GitHub Action that runs weekly to automatically update who's on rotation.

There's a lot of room for growth in this system, such as posting who's on rotation to Slack, generating a calendar viewable in Google Calendar or other calendar clients, automatically assigning PRs, extending to have rotations for different companies or project areas etc.

Signed-off-by: JP Simard <jp@jpsim.com>

### CODE REVIEW

File: .github/workflows/bump_support_rotation.yml
Comment (jpsim) [inline .github/workflows/bump_support_rotation.yml:23]: The `main` branch is protected so this may need to open a PR and assign the person on rotation.

Comment (jpsim) [reply .github/workflows/bump_support_rotation.yml:30]: Done in 9c50111.

File: docs/Support.md
Comment (buildbreaker) [inline docs/Support.md:5]:
```suggestion
issues and pull requests are handled in a timely manner from an Envoy Mobile maintainer.
```

Just because I'm an optimist: I think we can avoid using Lyft grouping so when our current friends are well onboarded and can help participate without being part of Lyft

File: .github/lyft_maintainers.yml
Comment (buildbreaker) [inline .github/lyft_maintainers.yml:1]: Maybe we can fold the maintainer role into this and we can better set expectations with our friends who want to be part of this -- wdyt?

Comment (jpsim) [reply .github/lyft_maintainers.yml:1]: I'm not sure I understand what you mean. Can you elaborate on what you mean by folding this maintainer role into this? This file _does_ define who the current support maintainer is.

File: docs/Support.md
Comment (jpsim) [reply docs/Support.md:5]: Some of the friends of Envoy Mobile have explicitly requested a rotation like this so they can reach someone within Lyft.

I'd love for there to be a more general Envoy Mobile rotation, and remove the need in having feedback from a specific company, at some point in the near future, at which point we can and should generalize the wording here.

**Review by Augustyniak — APPROVED** [review]
Nice 👌

**Comment by jpsim** [comment]
Can you please take a look @alyssawilk ?

File: docs/Support.md
Comment (alyssawilk) [inline docs/Support.md:28]: nit bee -> been

**Review by alyssawilk — COMMENTED** [review]
LGTM modulo minor nit.  I'm all for shipping and iterating :-)

**Review by alyssawilk — APPROVED** [review]

**Pull request merged**
